### DISCUSSION TITLE
Can we create user defined filters for mitigations

### DISCUSSION THREAD

**Discussion by rsrinivasanhome** [simonec73/threatsmanager#49] [Q&A] [answered]
Hi,
I had a query on filters in general .  Is  there any way to add custom filters without code changes.

In the DevOps extension how can we add filters ?
![image](https://user-images.githubusercontent.com/31912418/236123726-91575ff2-a44b-45e5-8e6f-8bdd06313be8.png)

**Comment by simonec73** [comment, marked as answer]
No, it's not possible. Those filters are defined in code as extensions.
